blackboardvc-portlet-webapp/src/main/java/org/jasig/portlet/blackboardvcportlet/mvc/sessionmngr/ViewSessionListController.java

```diff
@@ -153,13 +153,14 @@ public boolean matches(Object arg0) {
 			//Get information for all sessions
 		    final Set<SessionRecording> sessionRecordings = this.sessionDao.getSessionRecordings(session);
 		    recordings.addAll(sessionRecordings);
-		    
+
 		    //get launch URL
 		    sessionService.populateLaunchUrl(conferenceUser, session);
 		}
 		model.addAttribute("recordings", Ordering.from(SessionRecordingDisplayComparator.INSTANCE).sortedCopy(recordings));
-		
-		if(WindowState.MAXIMIZED.equals(request.getWindowState())) {
+
+		if(WindowState.MAXIMIZED.equals(request.getWindowState())
+				|| "exclusive".equals(request.getWindowState().toString())) {
```

Comment:
I would have done `WindowState.EXCLUSIVE` but fun fact, that isn't a standard portlet spec.